README.md

```diff
@@ -125,7 +132,7 @@ file of the accompanying Helm Chart.
 You can install the chart from the local filesystem.
 
 ```
-helm install https://github.com/lightbend/kafka-lag-exporter/releases/download/v0.5.4/kafka-lag-exporter-0.5.4.tgz
+helm install https://github.com/lightbend/kafka-lag-exporter/releases/download/v0.5.5/kafka-lag-exporter-0.5.5.tgz
```

Comment:
Do you mind reverting the changes to update the version for the example commands in the README?  I'd like it to reflect the current released version for people who browse the project homepage.  The release process will automatically update it.